api/user.go

```diff
@@ -588,8 +591,13 @@ func getAudits(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func createProfileImage(username string, userId string) ([]byte, *model.AppError) {
+var (
+	dpi      = flag.Float64("dpi", 72, "screen resolution in Dots Per Inch")
```

Comment:
These properties shouldn't come from flags.  It should be in the config.json if it needs to be configurable otherwise just choose the defaults.